### DISCUSSION TITLE
[Question] Does recently found xss-vulnerability affect old tinyMCE (<5) as well?

### DISCUSSION THREAD

**Discussion by vlwkaos** [tinymce/tinymce#8341] [General]
https://github.com/advisories/GHSA-gg8r-xjwq-4w92

The affected version states >= 6.0.0, < 6.3.1 AND < 5.10.7, so it assumes tinyMCE4 is affected as well?

Let me rephrase that: Is the affected code included in tinyMCEv4 as well?

because, afaik the related dialog code that can insert raw text does not exist in tinyMCEv4.

I know that the security policy states it does not support tinyMCE versions other than 5.10.x and 6.x. but it could be nice if I can get an answer. Thank you

**Comment by vlwkaos** [comment]
https://github.com/tinymce/tinymce/commit/9ee28e7c888f91d91c39d64495e49c3486909004#
this is related commit in v6

**Comment by lostkeys** [comment]
Hi @vlwkaos! We ended support for TinyMCE 4 two years ago and no longer provide _any_ updates to it. We haven't checked if the affected code is included in TinyMCE 4 either and we unfortunately can not give you any advice.  Our official answer is to always use a supported version of TinyMCE.
